Src/PerformanceCollector/Perf.Shared/Implementation/WebAppPerformanceCollector/WebAppPerformanceCollector.cs

```diff
@@ -153,7 +153,7 @@ private static double CollectCounter(string coutnerOriginalString, ICounterValue
                  throw new InvalidOperationException(
                     string.Format(
                         CultureInfo.CurrentCulture,
-                        Resources.WebAppPerformanceCounterReadFailed,
+                        "Failed to perform a read for web app performance counter {0}",
```

Comment:
Can you fix all the messages in a separate PR? Just to make everything consistent

Reply:
Yes will do it.